### ISSUE TITLE
Request: js_of_ocaml example

### ISSUE THREAD

**Issue by glennsl** [daypack-dev/timere#42]
It seems this has been made with js_of_ocaml, and platform independence more generally, in mind. But it's not at all clear to me how you would set up a js_of_ocaml project to actually use it. It certainly isn't sufficient to just install and use `timedesc`, as that will try to read files from the file system.

I can see there are various backends that can be used, but none that target js_of_ocaml specifically. There's also the `export-js-tzdb-*` executables which do target js_of_ocaml, but I don't see how to connect these with anything else. My guess based on this is that it might require using one of these executables to generate a js bundle of the timezone db, include that in your own code somehow, then create a custom backend to utilize it?

In short, would it be possible to provide an example or a more detailed description of how this could be accomplished?

**Comment by darrenldl** [comment]
Ah indeed I've largely forgotten about the js side of usage and it remains unpolished. One outdated example is at https://github.com/daypack-dev/timere-parse-demo

But indeed there remains a need for a proper guide.

**Comment by glennsl** [comment]
Thanks! So it seems the problem is not so much with the timezone database, but that there's no `tzlocal` backend for js. I don't think that should be too hard to implement though, so I'll see about doing that.

Also, a bit as an aside, I don't really need a tzdb. All I need, and I think this applies to most users, is the local timezone offset. I'll only ever deal with local time for interacting with the end-user, and UTC time for interacting with the server. Is it possible to only create a single fixed-offset time zone, get the current time for that, and deal only with that and UTC?

**Comment by darrenldl** [comment]
Not exactly advisable, but you can do so via picking tzlocal.none or tzlocal.utc backend, and supply your offset to all relevant calls via `Timedesc.Time_zone.make_offset_only[_exn]`.

Main issue is as soon as any extrapolation into future can cause error when you cross DST boundary. 

We experimented with dynamic lookup at here https://github.com/daypack-dev/doodlinator/blob/main/bookmarklet/crawler.ml#L24 which makes use of the tzdb-json files (granted they should be hosted in some CDN for production).
This is likely the better approach with lower space requirement and no compromise on correctness.

**Comment by glennsl** [comment]
True, that probably is a better approach. I'll look into that if it starts becoming a problem. Thanks for the guidance!
